release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -53,6 +53,12 @@ The installer now gathers serial console logs from the bootstrap and control pla
 
 For more information, see xref:../installing/installing-troubleshooting.adoc#installation-bootstrap-gather_installing-troubleshooting[Troubleshooting installation issues].
 
+[id="ocp-4-12-ironic-base-image-rhel-9"]
+==== Ironic container images use RHEL 9 base image
+In earlier versions of {product-title}, Ironic container images used Red Hat Enterprise Linux (RHEL) 8 as the base image. From {product-title} 4.12, Ironic container images use RHEL 9 as the base image. This is to enable support for CentOS Stream 9, Python 3.8, and Python 3.9 in Ironic components. 
+
+For further information about Ironic container images, see link:https://github.com/openshift/ironic-image[Ironic container image] and link:https://github.com/openshift/ironic-agent-image[Ironic agent container].
```

Comment:
Hi, is there a place in our documentation that you can link to rather than github? See our [contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites).

Reply:
:+1: on the GH comment. Also, I am not sure if there is specific guidance on the use of _further_, but I am most familiar with the simpler: "For more information..." in the docs.